website_sale_b2x_alt_price/readme/CONFIGURE.md

```diff
@@ -0,0 +1,10 @@
+To choose if you want to display or not the alternative price in the
+website products list, just enable or disable the feature from the
+"Customize" menu. This can be done independently per website:
+
+![toggle-list](../static/description/toggle-list.gif)
```

Comment:
You must update this configuration and its gif. It is now located in the "Customize" tab of the web editor.

Reply:
@pilarvargas-tecnativa I updated the Configure and added the new GIF. Please let me know your comments.